bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,298 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=80,
+                        max_line_length_affects_comments: bool=True,
+                        space_before_and_after_arrow: bool=False,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=0,
+                        spacing_in_empty_braces: int=0,
+                        camelcase_classes_name: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        check_whether_file_ends_in_newline: bool=False,
+                        tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                        no_tabs: bool=True,
+                        newlines_after_classes: bool=False,
+                        number_of_newlines_after_classes: int=3,
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=False,
+                        no_implicit_parentheses: bool=False,
+                        no_interpolation_in_single_quotes: bool=True,
+                        no_stand_alone_at: bool=True,
+                        disable_throwing_strings: bool=True,
+                        no_trailing_semicolons: bool=True,
+                        no_trailing_whitespaces: bool=True,
+                        no_trailing_whitespaces_affects_comments: bool=True,
+                        no_trailing_whitespaces_affects_empty_lines: bool=True,
+                        no_unnecessary_double_quotes: bool=False,
+                        use_english_operator: bool=False,
+                        spaces_around_operators: bool=False,
+                        space_after_comma: bool=True,
+                        cyclomatic_complexity: bool=True,
+                        cyclomatic_complexity_value: int=10,
```

Comment:
instead of offering the complexity and the value I'd just offer the value and let the user deactivate it somehow.

Also we should use the same setting name and convention for this for all bears, maybe we should generate a settings matrix or so? What do you think?